### PR TITLE
Replace base64.decodestring for base64.standard_b64decode

### PR DESCRIPTION
**Pull request by dechristo**
Fix deprecation warning from kube_config.py ([Issue #60](https://github.com/kubernetes-client/python-base/issues/60))

### CODE REVIEW

**Comment by yliaog** [comment]
thanks for the PR, could you please add the reasoning for the change in the description section of this PR?

**Comment by yliaog** [comment]
thanks for the description, could you please make "Issue #634" point to the issue?

**Comment by dechristo** [comment]
@yliaog Thanks for your attention. Done.

**Comment by yliaog** [comment]
could you make fixes in the test below too?
https://github.com/kubernetes-client/python-base/blob/master/config/kube_config_test.py

also Issue #634 points to issue 60, I'm curious where the number #634 come from?

File: config/kube_config_test.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
 
 
 def _base64(string):
-    return base64.encodestring(string.encode()).decode()
+    return base64.standard_b64decode(string.encode()).decode()
```
Comment (yliaog) [inline config/kube_config_test.py]: i guess you mean to use standard_b64encode

**Comment by yliaog** [comment]
no need for new PR, could you squash the two commits into one?

**Comment by dechristo** [comment]
@yliaog I did it. But one of the unit tests (test_data_given_file) failed. So I replaced   `base64.encodestring` for `base64.standard_b64encode` in kube_config.py

**Pull request merged**